python_modules/dagster/dagster/_cli/project.py

```diff
@@ -77,6 +77,40 @@ def scaffold_command(name: str):
     click.echo(_styled_success_statement(name, dir_abspath))
 
 
+@project_cli.command(
+    name="from-example",
+    help="Create a new Dagster project using one of the official Dagster examples.",
```

Comment:
Similar to the feedback I left on the other diff, I think we should clearly communicate what it means to create a new Dagster project - i.e. that it's creating a bunch of files and folders in a directory (not running some command against a remote Dagster instance).